docs/cli.md

```diff
@@ -0,0 +1,25 @@
+# Spectral CLI
+
+One installed, Spectral can be run via the command-line:
+
+```bash
+spectral lint petstore.yaml
+```
+
+Other options include:
+
+``` text
+  -e, --encoding=encoding      text encoding to use
+  -f, --format=json|stylish    formatter to use for outputting results
+  -h, --help                   show CLI help
+  -o, --output=output          output to a file instead of stdout
+  -q, --quiet                  no logging - output only
+  -r, --ruleset=ruleset        path to a ruleset file (supports remote files)
+  -s, --skip-rule=skip-rule    ignore certain rules if they are causing trouble
+  -v, --verbose                increase verbosity
+  --max-results=max-results    [default: all] maximum results to show
```

Comment:
max-results is deprecated, right?

Reply:
It's removed in https://github.com/stoplightio/spectral/pull/345